src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```
@@ -654,6 +712,18 @@ public LightningStrike strikeLightningEffect(Location loc)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public @Nullable Location findLightningRod(@NotNull Location location)
+	{
+		return null;
+	}
+
+	@Override
+	public @Nullable Location findLightningTarget(@NotNull Location location)
+	{
+		return null;
```

Should probably throw an exception